docs/concepts/async-await.mdx

```diff
@@ -0,0 +1,203 @@
+---
+title: "Async/Await"
+---
+
+TopK SDK provides full async/await support for the Python SDK as well as the Javascript SDK.
+
+## Python SDK
+
+`AsyncClient` is an asynchronous client that exposes a coroutine-based API, enabling non-blocking interaction with TopK through `asyncio`:
```

Comment:
`asyncio` is just one of python runtimes (like `tokio`), there are others. 

we can say something simpler like `AsyncClient` is an async version of the client that allows to use `async`/`await` keywords in python

Reply:
updated https://github.com/topk-io/topk/pull/233/commits/71d9a14ae1ddea5990f22d0a4a32745e511bdd33